### PR TITLE
Add cloud management solutions section and Brainboard

### PR DESCRIPTION
**Pull request by africanzoe**
Brainboard: solution to build and manage cloud infrastructures.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,6 +140,18 @@ Table of Contents
 
 **[⬆ back to top](#table-of-contents)**
 
+## Cloud management solutions
+  * [Brainboard](https://www.brainboard.co) - visually build and manage cloud infrastructures from end-to-end:
```
Comment (ripienaar) [inline README.md:144]: We do not add single entry catagories, either add this to an existing category - as one line - or move other matching items into this category

Comment (africanzoe) [reply README.md:144]: I moved Terraform.io into this category as well. Let me know please if it's OK.

Comment (ripienaar) [reply README.md:144]: Please make a one line description. Giant multi offering services like GitHub or huge clouds get a multi line. The rest just isn’t that varied.

Comment (africanzoe) [reply README.md]: Done.

**Comment by ripienaar** [comment]
Thnx

**Pull request merged**